bundles/mapping/toolbar/request/ShowMapMeasurementRequestHandler.ol.js

```diff
@@ -61,7 +61,6 @@ Oskari.clazz.define(
                 // if there is no content container, show the data in dialog
                 if (!me._dialogShown) {
                     dialog.show(me._title, '', me._buttons);
-                    dialog.makeDraggable();
```

Comment:
Remove draggable?

Reply:
Similar popups are not draggable (e.g. map marker, select features). Maybe I was thinking that I don't make another pr.